lib/Common/CommonDefines.h

```diff
@@ -101,7 +101,7 @@
 // ByteCode
 #define VARIABLE_INT_ENCODING 1                     // Byte code serialization variable size int field encoding
 #define BYTECODE_BRANCH_ISLAND                      // Byte code short branch and branch island
-
+#define ENABLE_UNICODE_API 1                        // Enable use of Unicode-related APIs
```

Comment:
@obastemur had mentioned that he thought it would be good to support a cross-plat build that didn't depend on the ICU. In that case, we'd want to set ENABLE_UNICODE_API = 0